packages/graphql_server/src/LinkedInScrapingService.ts

```diff
@@ -0,0 +1,136 @@
+import axios, { AxiosRequestConfig } from 'axios'
+import dotenv from 'dotenv'
+
+dotenv.config()
```

Comment:
@pczern is dotenv. okay to use?

Reply:
He recommended using dotenv in my other issue

Reply:
we are already loading env's @thomas-woehrle, see your .env.dev
any env you want, you have to notify Amin about

Reply:
@pczern I don't know what you mean with notifying Amin ?
I would suggest @jonassuess gets rid of dotenv.